lib/slack/message.rb

```diff
@@ -69,10 +75,20 @@ def send_message(channel)
           FDE::Slack::Notification.config.webhook,
           channel: channel,
           username: @username
-        )
+        ) do
+          http_client FDE::Slack::Util::HTTPClient
```

Comment:
@jugglinghobo Not sure where the method `http_client` is defined.

Reply:
This is how you configure the `http_client` that's used by the gem. I added a comment, see https://github.com/stevenosloan/slack-notifier#custom-http-client